CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+# 4.16.0 (2020-06-28)
+
+- Added a callback to the `addClick` to notify the component when a new item has been added to the data source. [#195](https://github.com/blackbaud/skyux-lookup/pull/195)
```

Comment:
```suggestion
- Added a callback to `addClick` to notify the component when a new item is added to the data source while the show more modal is open. [#195](https://github.com/blackbaud/skyux-lookup/pull/195)
```

Reply:
@blackbaud-paulah - the original issue was for things when the show more modal was open. However, the new callback isn't just for that and is for just when the data changes in general.

Reply:
@blackbaud-paulah - I have committed your suggestion.